source/guide_rust_server.rst

```diff
@@ -0,0 +1,111 @@
+.. author:: tpraxl <kontakt@macrominds.de>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/rust-logo-blk.svg
+      :align: center
+
+###########
+Rust Server
+###########
+
+It couldn't be simpler to run a Rust web server on Uberspace.
+
+Install Cargo, checkout your server code, install the server as a supervisord_ service and start it. Done.
+
+----
+
+.. note:: For this guide you might want to be familiar with the basic concepts of
+
+  * Rust_
+  * htaccess_
+  * supervisord_
+
+Prerequisites
+=============
+
+We need to install Rust_ first:
+
+::
+
+ [rust@taylor ~]$ curl https://sh.rustup.rs -sSf | sh
+ [rust@taylor ~]$ # Choose 1) for the default installation option that also sets up your PATH
+ [rust@taylor ~]$ source ~/.cargo/env
+
+Providing a Rust Server
+=======================
+
+To simplify this recipe, just follow the thruster-intro-tutorial_ to create your Hello World Server.
```

Comment:
Sorry but that won't work like this. We need to be able to adjust guides, all steps need to be documented in the lab.